apps/backend-e2e/src/map-review.e2e-spec.ts

```diff
@@ -331,6 +334,41 @@ describe('Map Reviews', () => {
           token: token
         }));
 
+      it('should 503 if the map-review key is true', async () => {
+        const [admin, adminToken] = await db.createAndLoginUser({
+          data: { roles: Role.ADMIN }
+        });
+
+        await req.patch({
+          url: 'admin/killswitch',
+          status: 204,
+          body: {
+            MAP_REVIEWS: true
+          },
+          token: adminToken
+        });
+
+        const response = await req.del({
+          url: `map-review/${review.id}`,
+          status: 503,
+          token: token
+        });
+
+        expect(response.body.statusCode).toEqual(503);
+
+        await req.patch({
+          url: 'admin/killswitch',
+          status: 204,
+          body: {
+            NEW_SIGNUPS: false,
+            RUN_SUBMISSION: false,
+            MAP_SUBMISSION: false,
+            MAP_REVIEWS: false
+          },
+          token: adminToken
+        });
```

Comment:
I don't like repeating this so frequently, better to have a dedicated function for it
[website-e865e35-feat(back-e2e) util for resetting killswitches.patch](https://github.com/user-attachments/files/15749302/website-e865e35-feat.back-e2e.util.for.resetting.killswitches.patch)
wasn't sure where to put it, here's a patch adding a tmp function to request.util until we move this system to redis.

Reply:
Remember to update this on all the other tests!